frontend/src/page/loginsignup.css

```diff
@@ -66,36 +66,6 @@ h1 {
   line-height: normal;
 }
 
```

Comment:
let's rename this `loginsignup.css` file to be `auth.css` since both Login and Signup are forms of authentication for the app.